spec/jobs/send_outgoing_email_job_spec.rb

```diff
@@ -19,4 +19,13 @@
       expect(outgoing_email.sent_at).to eq fake_time
     end
   end
+
+  describe '#job_object_id' do
```

Comment:
Why bother with this spec here? I'd rather see the two cases covered in `ApplicationJobSpec`.

Not a blocker.

Reply:
two reasons
* I made this before ApplicationJobSpec because I was lazy
* there is perhaps some value in testing two real implementations (one where the first arg is an integer, one where it is an object) rather than setting up fake classes that behave like that, in case the fake classes are too fake

Reply:
I still have my preference, but I see your point, and can totally live with this testing approach.